src/prob_causal_graph.jl

```diff
@@ -0,0 +1,47 @@
+using CausalInference, Distributions, LightGraphs, LinearAlgebra, Omega
+
+export prob_causal_graph, empirical_mechanism
+# Y = aX + b + c*u, u ~ N(0, 1)
+"""
+    `empirical_mechanism((; μ=..., Σ=...), A, B)`
+ Emprical mechanism of A given B
+"""
+function empirical_mechanism(P, A, B)
+    Z = P.Σ[A,B] * inv(P.Σ[B,B])
+    # (; μ = P.μ[A] + Z*(xB - P.μ[B]), Σ = P.Σ[A,A] - Z*P.Σ[B,A])
+    (; a=Z, b=P.μ[A] + Z * ( - P.μ[B]), c=P.Σ[A,A] - Z * P.Σ[B,A])
+end
+
+function prob_causal_graph(df; p=0.01, test=gausscitest)
```

Comment:
```suggestion
function prob_causal_graph(df, cg; p=0.01, test=gausscitest)
```
Better make `cg` an argument and provide a second method which calls it using `cg = pcalg(df, p, test)`